build/Dockerfile

```diff
@@ -1 +0,0 @@
-Dockerfile.amd64
```

Comment:
I think you incorrectly changed this from a symbolic link to `Dockerfile.amd64` to a file that just contains the text "Dockerfile.amd64"

Reply:
Oh thanks. I didn't understand why the contents looked the same but git detected a difference.